### PR TITLE
DOC(fatlabel.8.in): minor spelling typos

### LINKED ISSUE #145: Minor issue in man pages
Dear dosfstools maintainer,
the manpage-l10n project maintains a large number of translations of
man pages both from a large variety of sources (including dosfstools) as
well for a large variety of target languages.

During their work translators notice different possible issues in the
original (english) man pages. Sometimes this is a straightforward
typo, sometimes a hard to read sentence, sometimes this is a
convention not held up and sometimes we simply do not understand the
original.

We use several distributions as sources and update regularly (at
least every 2 month). This means we are fairly recent (some
distributions like archlinux also update frequently) but might miss
the latest upstream version once in a while, so the error might be
already fixed. We apologize and ask you to close the issue immediately
if this should be the case, but given the huge volume of projects and
the very limited number of volunteers we are not able to double check
each and every issue.

Secondly we translators see the manpages in the neutral po format,
i.e. converted and harmonized, but not the original source (be it man,
groff, xml or other). So we cannot provide a true patch (where
possible), but only an approximation which you need to convert into
your source format.

Finally the issues I'm reporting have accumulated over time and are
not always discovered by me, so sometimes my description of the
problem my be a bit limited - do not hesitate to ask so we can clarify
them.

I'm now reporting the errors for your project. If future reports
should use another channel, please let me know.

Man page: fatlabel.8
Issue: set → sets

"B<fatlabel> set or gets a MS-DOS filesystem label from a given device."

### PR DESCRIPTION
**Pull request by liketechnik**
Fixes #145

### CODE REVIEW

File: manpages/fatlabel.8.in [hunk 1]
```diff
@@ -23,7 +23,7 @@
 .\"
 .TH FATLABEL 8 @RELEASE_DATE@ "dosfstools @PACKAGE_VERSION@"
 .SH NAME
-fatlabel \- set or get MS-DOS filesystem label or volume ID
+fatlabel \- sets or get MS-DOS filesystem label or volume ID
```
Comment (elkablo) [inline manpages/fatlabel.8.in:26]: Looking at other man pages, for example `man ls` or `man diff`, they use first person / root form:
* `ls - list directory contents`
* `diff - compare files line by line`

so I would leave this as `set`.

**Review by elkablo — CHANGES_REQUESTED** [review]

**Comment by pali** [comment]
@liketechnik could you please fix above issue?

**Comment by liketechnik** [comment]
@pali I reverted the relevant change back. In case you prefer to have the change of this PR in a single commit, I'll happily squash the commits together.

**Comment by pali** [comment]
Please squash commits and rebase on top of the master branch.

**Comment by liketechnik** [comment]
@pali done.

**Pull request merged**